test/lib/analyzer/image-inspector.spec.ts

```diff
@@ -75,6 +75,11 @@ describe("extractImageDetails", () => {
       imageName: "foo/bar",
       tag: "sha256:8756a25c4c5e902c4fe20322cc69d510a0517b51eab630c614efbd612ed568bf",
     }}
+    ${"localhost:1234/foo/bar:alpine3.15@sha256:8756a25c4c5e902c4fe20322cc69d510a0517b51eab630c614efbd612ed568bf"} | ${{
+      hostname: "localhost:1234",
+      imageName: "foo/bar",
+      tag: "sha256:8756a25c4c5e902c4fe20322cc69d510a0517b51eab630c614efbd612ed568bf",
```

Comment:
comment for a future discussion:

technically this is not a `tag` - it's a `digest`. should we model this differently in the future?

Reply:
Yes, that is right. I think the terminology is not consistent everywhere.